libres/lib/enkf/analysis_config.cpp

```diff
@@ -252,7 +252,14 @@ void analysis_config_add_module_copy(analysis_config_type *config,
 analysis_module_type *
 analysis_config_get_module(const analysis_config_type *config,
                            const char *module_name) {
-    return config->analysis_modules.at(module_name);
+    if (analysis_config_has_module(config, module_name)) {
+        return config->analysis_modules.at(module_name);
+    } else {
+        char buffer[50];
+        sprintf(buffer, "Internal error - Analisys module named %s not found",
+                module_name);
+        throw std::invalid_argument(buffer);
```

Comment:
If you `#include <fmt/format.h>` you can do:

```suggestion
        throw std::invalid_argument(fmt::format("Analysis module named {} not found", module_name));
```

As it stands currently, this code is susceptible to buffer overflow.

Reply:
nice!